src/style/app-fixed.module.css

```
@@ -5764,9 +5765,10 @@ form {
   overflow: hidden;
   -webkit-box-orient: vertical;
   -webkit-line-clamp: var(--max-lines);
-
+  line-clamp: var(--max-lines);
   font-size: 1.3rem !important;
   font-weight: 600;
+  line-clamp: var(--max-lines);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

line-clamp is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)